.github/workflows/python-test-suite.yml

```diff
@@ -58,7 +58,15 @@ jobs:
 
       - name: Run the python tests suite
         working-directory: python
-        run: uv run --python ${{ matrix.python-version }} pytest tests -m "not slow"
+        run: |
+          uv run --python ${{ matrix.python-version }} pytest tests -m "not slow" --cov=src/magika --cov-report=xml --cov-report=term
```

Comment:
Is this not already covered by the change in pytest.ini?

Reply:
The configuration in pytest.ini centralizes the coverage settings, but the run command ensures the CI pipeline executes pytest correctly. However, I can remove the redundant options from the YAML file to avoid duplication.